h2o-genmodel/src/main/java/hex/genmodel/easy/EasyPredictModelWrapper.java

```diff
@@ -21,6 +21,7 @@
 import java.awt.image.BufferedImage;
 import java.io.*;
 import java.net.URL;
+import java.security.InvalidParameterException;
```

Comment:
This kind of exception shouldn't be used, use IllegalArgumentException instead

Reply:
Done!